ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -428,10 +428,9 @@ objects:
         cp -r /var/lib/openshift-install/upi/metal/* /tmp/artifacts/terraform/
         cp /bin/terraform-provider-matchbox /tmp/artifacts/terraform/
 
-        PXE_INITRD_URL="$(jq -r '.baseURI + .images["initramfs"].path' /var/lib/openshift-install/rhcos.json)"
-        PXE_KERNEL_URL="$(jq -r '.baseURI + .images["kernel"].path' /var/lib/openshift-install/rhcos.json)"
+        PXE_INITRD_URL="$(jq -r '.baseURI + (.images["live-initramfs"].path // .images["initramfs"].path)' /var/lib/openshift-install/rhcos.json)"
+        PXE_KERNEL_URL="$(jq -r '.baseURI + (.images["live-kernel"].path // .images["kernel"].path)' /var/lib/openshift-install/rhcos.json)"
         # 4.3 unified 'metal-bios' and 'metal-uefi' into just 'metal'
-        PXE_OS_IMAGE_URL="$(jq -r '.baseURI + (.images["metal-bios"].path // .images["metal"].path)' /var/lib/openshift-install/rhcos.json)"
```

Comment:
If we want this code to also work for old OpenShift versions, we need to keep this bit right?

Reply:
Yes, only interested in testing this out right now to figure out if I need to make changes installer repo to conditionalize emitting the kernel arg, what happens if coreos.inst.image_url is set to an empty string?

Reply:
@sdodson https://github.com/coreos/coreos-installer/blob/d2ae5a16d2dbbb299d6fa443734b2c507b28c97e/README.md#kernel-command-line-options-for-coreos-installer-running-as-a-service might be helpful

Reply:
Hmm...it *might* be true that an empty `coreos.inst.image_url` is the same as "unset", but I wouldn't rely on it; we should change the job to only set that variable on <= 4.5 installs.